datasets/Met-Office-Regional-average-climate-observations-UK-annual-rainfall-2021/main.py

```diff
@@ -0,0 +1,17 @@
+import pandas as pd 
+from gssutils import *
+from csvcubed.models.cube.qb.catalog import CatalogMetadata
+from datetime import datetime
+
+df = pd.read_csv("raw.csv")
+
+df['Year'] = df['Year'].astype(str).astype(int)
+
+df.to_csv('observations.csv', index=False)
+catalog_metadata = CatalogMetadata(
+    title = "Regional average climate observations uk annual rainfall 2021",
```

Comment:
Looking at the older version of this dataset and the title use I would change the title to be: 
`title = Regional annual average rainfall with trends 1836 - 2021`

Reply:
For the title, I thought we must use that provide by info.json from the Airtable against that of last year. Thanks for the observation